libs/au-pair-dashboard/feature/src/lib/parent-rating-modal/parent-rating-modal.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="modal-body">
+    <div class="heading ">
+        <ion-row>
+            <ion-col size="11" class="ion-text-center act">Parent Rating</ion-col>
```

Comment:
For Title to be properly centered please put 

size="10" offset = "1" in the Parent Rating ion-col